styles/bricks-common.css

```diff
@@ -19,11 +19,6 @@ h3 {
   letter-spacing: -0.04em;
 }
 
-h1, h2, h3 {
```

Comment:
this looks accidental (it was duplicated and one of them was already removed in another PR), did you just intend to prevent duplication?

Reply:
Yes. I think linter was complaining about.